assets/queries/k8s/allowed_capabilities_sys_admin/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "allowed_capabilities_sys_admin",
+  "queryName": "Allowed Capabilities Sys Admin",
+  "severity": "MEDIUM",
+  "category": null,
+  "descriptionText": "Containers should not have CAP_SYS_ADMIN Linux capability",
+  "descriptionUrl": "https://jamesdefabia.github.io/docs/user-guide/containers/"
```

Comment:
We should point to kubernetes official documentation if appliable. If not, leave empty

Reply:
I did not find a reference to "sys_admin" in the kubernetes official documentation. But I will point to the security context of the kubernetes official documentation

Reply:
You could even be more specific in the section on that page, where [capabilities](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-capabilities-for-a-container) are mentioned. Right on the first line the is a link to manpages about [Linux capabilities](https://man7.org/linux/man-pages/man7/capabilities.7.html), which CAP_SYS_ADMIN is part of.